recipe/conda_build_config.yaml

```diff
@@ -16,3 +16,4 @@ version:
   - 12.1
   - 12.2
   - 12.3
+  - 12.4
```

Comment:
Should this list be updated to reflect what we will actually support?

Reply:
Although I'm not super opinionated either way.

Reply:
fair enough